dali/kernels/common/utils.h

```diff
@@ -58,6 +59,16 @@ OutShape GetStrides(const Shape& shape) {
   return strides;
 }
 
+inline int64_t GetSMCount() {
```

Comment:
This function should go to `cuda_utils`, right next to `MaxThreadsPerBlock`.

Reply:
Moved to `cuda_utils.h` and changed to `int`